eotsmanager/cmd/eotsd/daemon/keys.go

```diff
@@ -50,53 +50,84 @@ func NewKeysCmd() *cobra.Command {
 		}
 
 		cmd.SetOut(oldOut)
-		return saveKeyNameMapping(cmd, args)
+		keyName := args[0]
+		eotsPk, err := saveKeyNameMapping(cmd, keyName)
+		if err != nil {
+			return err
+		}
+
+		return printFromKey(cmd, keyName, eotsPk)
 	}
 
+	saveKeyOnPostRun(keysCmd, "import")
+	saveKeyOnPostRun(keysCmd, "import-hex")
```

Comment:
let's also add `rename` here

Reply:
we currently have a check to avoid duplicated eots pk 
https://github.com/babylonlabs-io/finality-provider/blob/27349c94ac8b5dc526813c71ad18bed67b8a95be/eotsmanager/store/eotsstore.go#L65

I will merge this one as it is and we can think on how to solve this one